views/apartment.erb

```diff
@@ -0,0 +1,11 @@
+<!DOCTYPE html>
```

Comment:
This belongs in app/views/layout.erb.  The views for each model are just html within the `<body>`.